### PR TITLE
FAT-742 added job profile for import file with creating of the new instance, holding and item

### PR DESCRIPTION
**Pull request by TetianaParanich**

### CODE REVIEW

File: cypress.json [hunk 1]
```diff
@@ -1,12 +1,13 @@
...
+  
+  "baseUrl": "https://folio-snapshot-load.dev.folio.org",
   "viewportWidth": 1920,
   "viewportHeight": 1080,
   "video": false,
   "defaultCommandTimeout": 21000,
   "pageLoadTimeout": 120000,
   "env": {
-    "OKAPI_HOST": "https://folio-snapshot-okapi.dev.folio.org",
+    "OKAPI_HOST": "https://folio-snapshot-load-okapi.dev.folio.org",
```
Comment (dmitriyfolio) [inline cypress.json:10]: revert please to initial state

File: cypress.json [hunk 2]
```diff
@@ -1,12 +1,13 @@
 {
-  "baseUrl": "https://folio-snapshot.dev.folio.org",
+  
+  "baseUrl": "https://folio-snapshot-load.dev.folio.org",
```
Comment (KyryloBrener) [inline cypress.json:3]: Please revert this change

File: cypress.json (hunk 1 above)
Comment (KyryloBrener) [inline cypress.json:10]: Please revert this change

File: cypress/support/fragments/data_import/action_profiles/actionProfiles.js [hunk 3]
```diff
@@ -0,0 +1,22 @@
+import { getLongDelay } from '../../../utils/cypressTools';
+import { Button } from '../../../../../interactors';
+
+export default class ActionProfiles {
```
Comment (KyryloBrener) [inline cypress/support/fragments/data_import/action_profiles/actionProfiles.js:4]: Probably let's not use classes and rather follow our common approach with default export?

File: cypress/integration/data_import/upload-file-create.spec.js [hunk 4]
```diff
@@ -0,0 +1,82 @@
...
+    JobProfiles.selectJobProfile(specialJobProfile.profileName);
+
+    JobProfiles.runImportFile();
+    cy.wait(10000);
+    Logs.searchJobProfileByName(DataImport.uniqueFileName);
+    Logs.waitUntilSearchJobProfile();
+    Logs.checkStatusOfJobProfile();
+    Logs.selectJobProfile();
+    Logs.checkCreatedItems();
+  });
```
Comment (KyryloBrener) [inline cypress/integration/data_import/upload-file-create.spec.js:80]: Please add TODO for deletion of created data. And also resolve conflicts : )
